### PR TITLE
Modified AppNet Example to use Alamofire 

### PR DESCRIPTION
**Pull request by henryhardy**
Modified Appnet example to use alamofire. added alamofire to podfile.

### CODE REVIEW

File: AppNet/Networking.swift [hunk 1]
```diff
@@ -1,29 +1,59 @@
 import UIKit
 import DATAStack
 import Sync
+import Alamofire
 
 class Networking: NSObject {
     let AppNetURL = "https://api.app.net/posts/stream/global"
     let dataStack: DATAStack
-
+
```
Comment (3lvis) [inline AppNet/Networking.swift]: Could you remove this whitespaces?

File: AppNet/Networking.swift [hunk 2]
```diff
@@ -1,29 +1,59 @@
...
 class Networking: NSObject {
     let AppNetURL = "https://api.app.net/posts/stream/global"
     let dataStack: DATAStack
-
+    
     required init(dataStack: DATAStack) {
         self.dataStack = dataStack
     }
-
+
```
Comment (3lvis) [inline AppNet/Networking.swift]: Could you remove this whitespaces?

File: AppNet/Networking.swift [hunk 3]
```diff
@@ -34,23 +64,23 @@ class Networking: NSObject {
...
         }
     }
-
+    
     func changeNotification(notification: NSNotification) {
         let updatedObjects = notification.userInfo?[NSUpdatedObjectsKey]
         let deletedObjects = notification.userInfo?[NSDeletedObjectsKey]
         let insertedObjects = notification.userInfo?[NSInsertedObjectsKey]
-
+
```
Comment (3lvis) [inline AppNet/Networking.swift]: Could you remove this whitespaces?

File: AppNet/Networking.swift [hunk 4]
```diff
@@ -1,29 +1,59 @@
...
+                }
+        }
+        
+        
+        
+        //OLD CODE
+        
+        //        
+        //        let session = NSURLSession.sharedSession()
+        //        let request = NSURLRequest(URL: NSURL(string: AppNetURL)!)
```
Comment (3lvis) [inline AppNet/Networking.swift]: Also, please remove the commented code.

File: AppNet/Networking.swift [hunk 5]
```diff
@@ -1,29 +1,59 @@
...
-        session.dataTaskWithRequest(request, completionHandler: { data, response, error in
-            if let data = data, json = (try? NSJSONSerialization.JSONObjectWithData(data, options: [])) as? [String: AnyObject] {
-                Sync.changes(json["data"] as! Array, inEntityNamed: "Data", dataStack: self.dataStack, completion: { error in
+        
+        // ALAMOFIRE CODE
+        
+        Alamofire.request(.GET, AppNetURL)
+            .responseJSON { response in
+                print(response.request)  // original URL request
+                print(response.response) // URL response
```
Comment (3lvis) [inline AppNet/Networking.swift]: Could you remove this logs?

File: AppNet/Networking.swift [hunk 6]
```diff
@@ -1,29 +1,59 @@
...
-
+    
     func fetchItems(completion: (NSError?) -> Void) {
-        let session = NSURLSession.sharedSession()
-        let request = NSURLRequest(URL: NSURL(string: AppNetURL)!)
-        session.dataTaskWithRequest(request, completionHandler: { data, response, error in
-            if let data = data, json = (try? NSJSONSerialization.JSONObjectWithData(data, options: [])) as? [String: AnyObject] {
-                Sync.changes(json["data"] as! Array, inEntityNamed: "Data", dataStack: self.dataStack, completion: { error in
+        
+        // ALAMOFIRE CODE
```
Comment (3lvis) [inline AppNet/Networking.swift]: Please remove this comment too.

**Comment by 3lvis** [comment]
Hi @henryhardy,

Thanks for this PR! I added a few comments. I hope it's ok for you.

 :sparkles: :heart: :sparkles:

**Comment by henryhardy** [comment]
Thanks man. Will sort out and update =]

**Comment by 3lvis** [comment]
I'm going to clean up this myself :)

**Comment by 3lvis** [comment]
Done https://github.com/hyperoslo/Sync/commit/c454f91a2376d941a09ecc2e355adb8c741ad90d!

**Pull request merged**